README.md

```diff
@@ -198,95 +293,4 @@ This repo follows [semantic versioning](https://semver.org/). Here are some exam
-
-**For Design System Team only:**
-
-1. If you are unsure if a new release should be created, check with the Release Manager and/or the rest of the team first, to make sure it's worth the effort at this time.
-2. Get the version number from [`packages/core/package.json`](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/core/package.json#L4), ensuring it's up-to-date and new. If the core, web-components, and/or react-components packages versions haven't been updated yet, submit a PR to update them prior to performing the release, or things won't work correctly with the automation.
-3. From the [repo's homepage](https://github.com/department-of-veterans-affairs/component-library) click on "Releases" in the right-hand sidebar.
-4. Click on the "Draft a new release" button near the top of the page.
-5. Click on the 'Choose a tag' drop-down and type the letter `v` followed by the new "core" version number (should look like `v16.1.0`). The target should remain `main`.
-6. For the release title, type the same thing you entered for the tag (`v{versionNumber}`).
-7. Click on the "Generate release notes" button. If the button is disabled, double-check that the tag/version number is correct and hasn't been released before.
-8. Review the release notes for any typos and/or unneeded notes. Remember that these release notes are intended for public use, so they should be professional in their tone and appearance.
-9. Take a screenshot of the release notes and post in the `#platform-design-system-team` channel on slack; ask for others to double-check that everything looks good and that there aren't any last-minute additions to the release that need to be included.
-10. Back in GitHub, ensure the "Set as the latest release" checkbox is checked.
-11. Press the "Publish release" button. GitHub Actions will take care of any necessary build and publishing steps.
-12. If [vets-website](https://github.com/department-of-veterans-affairs/vets-website) will need to take advantage of the latest release sooner than later, open a PR to update the dependency version there (update your local copy and then submit a PR to merge the latest version bump).
-13. Finally, go to your local copy of the [vets-design-system-documentation](https://github.com/department-of-veterans-affairs/vets-design-system-documentation) repo and update the version requirement, submitting a PR for that as well.
-
-# Running Build via Storybook
-
-1. `cd packages/web-components/`
-   1. `yarn install`
-   2. `yarn build`
-   3. `yarn build-bindings` (build React bindings)
-   4. `yarn watch:stencil` (optional)
-2. `cd ../react-components/`
-   1. `yarn install`
-   2. `yarn build`
-3. `cd ../core/`
-   1. `yarn install`
-   2. `yarn build`
-4. `cd ../storybook/`
-   1. `yarn install`
-   2. `yarn storybook`
-
-This will allow you to run Storybook locally to view all components
-
-# Stencil Dev Server for Testing IE11
```

Comment:
I'm actually removing the **Stencil Dev Server for Testing IE11** section because it seems pretty outdated. We don't support IE11 anymore so there shouldn't be a need to test there.